admin/edit-user.html

```diff
@@ -3,10 +3,112 @@
+        </div>
+      </div>
+      <div class="flex flex-wrap items-end">
+        <div class="w-100 w-50-ns mb4 pr1-ns">
+          <label for="email" class="db mb2 fw6">
+            Email *
+          </label>
+          <input id="email" type="text" class="w-100 pa3 br2 b--black-10" />
+        </div>
+      </div>
+      <div class="flex flex-wrap items-end">
+        <div class="w-100 w-50-ns mb4 pr1-ns">
+          <label for="role" class="db mb2 fw6">
+            Role
+          </label>
+          <ul class="list pl0 ml0 center mw6 ba bw1 b--black-10 br2">
+            <li class="ph3 pv3 bb bw1 b--black-10">
+              <div class="pb2">
+                <input
+                  type="radio"
+                  id="search"
+                  name="role"
+                  value="search"
+                  class="v-top"
+                  checked
+                />
+                <label for="search" class="pl1 fw6">Search</label>
+              </div>
+              <div>
+                <ul class="pl3">
+                  <li class="mb1">&bull;&nbsp;Can search records</li>
+                </ul>
+              </div>
+            </li>
+            <li class="ph3 pv3">
+              <div class="pb2">
+                <input
+                  type="radio"
+                  id="admin"
+                  name="role"
+                  value="admin"
+                  class="v-top"
+                />
+                <label for="admin" class="pl1 fw6">Admin</label>
+              </div>
+              <div>
+                <ul class="pl3">
+                  <li class="mb1">&bull;&nbsp;Can search records</li>
+                  <li class="mb1">&bull;&nbsp;Can manage users and groups</li>
+                </ul>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="flex flex-wrap items-end">
+        <div class="w-100 w-50-ns mb4 pr1-ns">
+          <label for="group" class="db mb2 fw6">
+            Group
+          </label>
+          <select
+            id="group"
+            class="w-100 pa3 br2 bw1 b--black-10 input-reset bg-white black-60 indent"
+          >
+            <option value="">
+              -- Please select a group --
+            </option>
+            <option value="option1">Option 1</option>
+            <option value="option2">Option 2</option>
+            <option value="option3">Option 3</option>
+          </select>
+        </div>
+      </div>
+      <div class="flex flex-wrap items-end">
+        <div class="w-100 w-50-ns mb4 pr1-ns">
+          <ul>
+            <li>
+              New users will get an email notification with a link to set their
+              password.
+            </li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="flex flex-wrap items-end">
+        <div class="w-100 w-50-ns mb3">
+          <button
+            class="br2 bg-blue white db w-100 tc pv3 btn--search"
```

Comment:
The button is missing the hover styles, please see the design system.

And for the purposes of this prototype, let's just make it an `<a href` and link back to the admin view. We'll want to actually use a button when it's implemented in the real repo though.

Reply:
Thanks for the review @hmarcks. Edits added save for the select styling.

Reply:
Button updated to <a>